src/Modal.js

```diff
@@ -466,4 +485,7 @@ Modal.Header = Header;
 Modal.Title = Title;
 Modal.Footer = Footer;
 
+Modal.TRANSITION_DURATION = 300;
+Modal.BACKDROP_TRANSITION_DURATION = 150;
```

Comment:
No.. I just thought about public api and documentation.

--
Anyway as I said it already - this is not essential.

It is more because of my curiosity and my wish to learn all the things.
And now - because you clarified it - I know that it is configurable in fact.
I just hadn't given it more thought in the first place :blush: